### ISSUE TITLE
Debian packaging improvements and toolbar styles

### ISSUE THREAD

**Issue by clefebvre** [maoschanz/drawing#112]
Hi Romain,

I made a couple of changes on the 0.4 branch:

https://github.com/linuxmint/drawing/commit/ea55e52f287d1b09f349ff0bae4f67820ccebaf1

The version in debian/changelog was still at 0.4.6 despite the version bump, this bumps it to 0.4.7.

https://github.com/linuxmint/drawing/commit/1e4cf5af068399a68656c9baecbc9e1cc7ccac07

Debian/control specifies architecture: any. This changes it to architecture: all, so we end up with an _all.deb that works everywhere, without needing to compile Drawing separately for each architecture.

https://github.com/linuxmint/drawing/commit/58f734c798cb722bdb5464d1c7ed259882d186cb

Adds the .primary-toolbar class name to the toolbars CSS to make them look like proper toolbars. Here's an example of a theme for which this makes a difference:

Before:

![image](https://user-images.githubusercontent.com/1138515/68127391-b5b97a00-ff0d-11e9-96a2-4f2d4f4418cd.png)

After:

![image](https://user-images.githubusercontent.com/1138515/68127351-a0445000-ff0d-11e9-9cac-b046b584cfae.png)

I wanted to run these through you before submitting PRs. Let me know if you want more info, and if you want PRs for these, I'd be happy to get them ready.

Thanks.

**Comment by clefebvre** [comment]
On the toolbar commit, all that is needed is https://github.com/linuxmint/drawing/commit/58f734c798cb722bdb5464d1c7ed259882d186cb#diff-c3ea45112661f34debbceff8c3d55881R226. I just noticed glade modified way more than that. I can reimplement that much more cleanly using a text editor.

**Comment by maoschanz** [comment]
interesting, i didn't even know there was a style class for this (with Adwaita the difference isn't very visible)

The debian files are not up-to-date at all because i'm not managing the PPA myself, i don't even have the build dependencies installed on my current laptop and i know almost nothing about debian packaging. If it works fine with "true debian" too, yes you can submit a PR on 0.4

----

also, I've a question concerning Cinnamon, maybe you can answer it: the actions available with a right-click on the launcher have icons, where are they defined?

**Comment by clefebvre** [comment]
Good question :) I had to look up the Cinnamon source code.. they're not part of the AppInfo spec, they're hardcoded by the applet @ https://github.com/linuxmint/cinnamon/blob/master/files/usr/share/cinnamon/applets/grouped-window-list%40cinnamon.org/constants.js#L14.

The debian changes work fine in any Debian system.

**Comment by clefebvre** [comment]
![image](https://user-images.githubusercontent.com/1138515/68157745-2bd9d300-ff46-11e9-8bf3-6ff5c6967654.png)

```
[Desktop Entry]
Name=Drawing
Comment=Simple drawing utility
Exec=drawing %U
# Icon name, do not translate
Icon=com.github.maoschanz.drawing
Terminal=false
Type=Application
StartupNotify=true
# This is a list of keywords. "Paint" should not be translated, the others should. Don't forget the semicolons.
Keywords=Paint;Sketch;Pencil;
Categories=Graphics;GNOME;GTK;
MimeType=image/png;image/bmp;image/jpeg;
Actions=new-window;draw;screen-shot;

[Desktop Action new-window]
Name=New Window
Exec=drawing --new-window

[Desktop Action draw]
Name=New Image
Exec=drawing --new-tab

[Desktop Action screen-shot]
Name=Edit Image in Clipboard
Exec=drawing --edit-clipboard

```

It's not ideal but that's how it works at the moment. If you want us to add new icons/names or change the way it works, don't hesitate to suggest ideas.

**Comment by clefebvre** [comment]
I opened PRs on 0.4 for the changes. I'll close this issue now.

**Comment by maoschanz** [comment]
oh... :/ sad it's not in the specs

now that i have that list i'll try to see which ones are the best

**Comment by maoschanz** [comment]
At first I assumed the "fix" to issue #55 looked like shit on Ubuntu because of their GTK theme, but here this is Mint-Y, which i have installed. And my test looks far better than your screenshot
![Capture d’écran de 2019-11-05 01-13-28](https://user-images.githubusercontent.com/19996113/68168619-ad8f2800-ff69-11e9-9271-4a7797a9502c.png)
 so i guess it's a font issue? What is kind of font available by default should i [write as an alternative to Cantarell](https://github.com/maoschanz/drawing/blob/db7029eb891377c913bbf542a1e7c61cf1240123/src/icons/unit-pixels-symbolic.svg#L5)?

**Comment by maoschanz** [comment]
at work i installed mint, and noticed that it's the version of drawing where the size units are written as text with an ugly font instead of a true icon. Is it compatible with mint policies to change the version of a package to fix minor details? 

(the version 0.4.8 actually changed a ton of other small things, but this issue in particular looks very bad imo)
